core/graph/src/test/java/uk/gov/gchq/gaffer/graph/GraphTest.java

```diff
@@ -1438,7 +1438,7 @@ public void shouldAddHookAndGetGraphHooks() throws Exception {
     }
 
     @Test
-    public void shouldAddNamedOperationResolverHookFirst() throws Exception {
+    public void shouldAddNamedOperationResolverHookAfterNamedOperationResolver() throws Exception {
```

Comment:
Minor comment, but I think this method name should be:
shouldAddNamedViewResolverHookAfterNamedOperationResolver